src/domain/spaces/address-books/entities/address-book-item.entity.ts

```diff
@@ -1,27 +1,12 @@
-import { RowSchema } from '@/datasources/db/v2/entities/row.entity';
-import type { Space } from '@/domain/spaces/entities/space.entity';
-import { SpaceSchema } from '@/domain/spaces/entities/space.entity';
 import { AddressSchema } from '@/validation/entities/schemas/address.schema';
 import { NameSchema } from '@/domain/common/entities/name.schema';
 import { z } from 'zod';
 
-// We need explicitly define ZodType due to recursion
-export const AddressBookItemSchema: z.ZodType<
-  z.infer<typeof RowSchema> & {
-    space: Space;
-    chainIds: Array<string>;
-    address: `0x${string}`;
-    name: string;
-    createdBy: `0x${string}`;
-    lastUpdatedBy: `0x${string}`;
-  }
-> = RowSchema.extend({
-  space: z.lazy(() => SpaceSchema),
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Nit: we can delete this.

Reply:
Its only being used as a type so the linter complains. Alternatively, we could export it even if its not used anywhere.